pfcwd/main.py

```diff
@@ -162,5 +169,34 @@ def stop(ports):
             continue
         configdb.mod_entry("PFC_WD_TABLE", port, None)
 
+# Set WD default configuration on server facing ports when enable flag is on
+@cli.command()
+def start_default():
+    """ Start PFC WD by default configurations  """
+    configdb = swsssdk.ConfigDBConnector()
+    configdb.connect()
+    enable = configdb.get_entry('DEVICE_METADATA', 'localhost').get('default_pfc_wd_status')
+    if enable != "enable":
+       return
+    device_type = configdb.get_entry('DEVICE_METADATA', 'localhost').get('type')
+    if device_type != "ToRRouter":
+        return
+    port_num = len(configdb.get_table('PORT').keys())
+    vlan_members = [p[1] for p in configdb.get_table('VLAN_MEMBER').keys()]
+
+    pfcwd_info = {
+        'detection_time': DEFAULT_DETECTION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
+        'restoration_time': DEFAULT_RESTORATION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
```

Comment:
let's not do the calculation here, just use the DEFAULT_DETECTION_TIME. The reason is that the detection time is based on external network requirement, it should not be set based on the port num in the system.